test/Microsoft.AspNetCore.Mvc.WebApiCompatShimTest/DefaultContentNegotiatorTest.cs

```diff
@@ -149,10 +149,13 @@ public class DefaultContentNegotiatorTests
         {
             get
             {
-#if !NETCOREAPP1_1
+#if NET46
                 // Only mapping and accept makes sense with q != 1.0
                 MediaTypeFormatterMatch matchMapping10 = CreateMatch(1.0, MediaTypeFormatterMatchRanking.MatchOnRequestWithMediaTypeMapping);
                 MediaTypeFormatterMatch matchMapping05 = CreateMatch(0.5, MediaTypeFormatterMatchRanking.MatchOnRequestWithMediaTypeMapping);
+#elif NETCOREAPP2_0
+#else
+#error the target framework needs to be updated.        
```

Comment:
What is the point of including this boilerplate multiple times in the same file?